packages/firebase/externs/firebase-auth-externs.js

```diff
@@ -1043,7 +1043,59 @@ firebase.auth.Auth.prototype.useDeviceLanguage = function() {};
 
 /**
  * Creates a new user account associated with the specified email address and
- * password.
+ * password and returns any additional user info data or credentials.
+ *
```

Comment:
Add another paragraph.

This method will be renamed to `createUserWithEmailAndPassword` replacing the existing method with the same name in the next major version change.

Do the same for the other 3.